tests/transforms/test_rand_simulate_low_resolution.py

```
@@ -78,6 +81,47 @@ def test_value(self, arguments, image, expected_data):
+        errors = []
+        expected_track_meta = get_track_meta()
+        start_barrier = threading.Barrier(8)
+
+        def run_transform():
+            img = torch.ones(1, 4, 4, 4)
+            tfm = RandSimulateLowResolution(prob=1.0, zoom_range=(0.5, 0.6))
+            start_barrier.wait()  # synchronise so all threads hammer the transform at once
+            try:
+                for _ in range(50):
+                    tfm(img)
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)